### ISSUE TITLE
Flowchart multi directional arrows not working with asymmetrical shape

### ISSUE THREAD

**Issue by jwatkin** [mermaid-js/mermaid#1732]
flowchart LR
    %% A is an asymmetrical shape.
    A>Test];
    A <--> B;
    %% The code above unexpectedly renders like this: Test --> B. The expectation is that it should render like this: Test <--> B. 

    %% C is a box shape.
    C[Test];
    C <--> D;
    %% The code above works as expected and renders like this: Test <--> D.

**Comment by jwatkin** [comment]
Some other shapes break multi directional arrows too.

**Comment by knsv** [comment]
Thanks for reporting!

**Comment by patrickbrett** [comment]
Has this been fixed? Looking in the live editor and can't reproduce the issue:

<img width="190" alt="Screen Shot 2021-06-27 at 3 32 07 pm" src="https://user-images.githubusercontent.com/6735055/123533955-d5de2280-d75c-11eb-939c-8e9b597d9009.png">

```
flowchart LR
    A>Test];
    A <--> B;

    C[Test];
    C <--> D;
```

**Comment by tbertran** [comment]
> Has this been fixed?

@patrickbrett  No, it still breaks if you [use `graph`](https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb2RlIjoiZ3JhcGggTFJcbiAgICAlJSBBIGlzIGFuIGFzeW1tZXRyaWNhbCBzaGFwZS5cbiAgICBBPlRlc3RdO1xuICAgIEEgPC0tPiBCO1xuICAgICUlIFRoZSBjb2RlIGFib3ZlIHVuZXhwZWN0ZWRseSByZW5kZXJzIGxpa2UgdGhpczogVGVzdCAtLT4gQi4gVGhlIGV4cGVjdGF0aW9uIGlzIHRoYXQgaXQgc2hvdWxkIHJlbmRlciBsaWtlIHRoaXM6IFRlc3QgPC0tPiBCLlxuXG4gICAgJSUgQyBpcyBhIGJveCBzaGFwZS5cbiAgICBDW1Rlc3RdO1xuICAgIEMgPC0tPiBEO1xuICAgICUlIFRoZSBjb2RlIGFib3ZlIHdvcmtzIGFzIGV4cGVjdGVkIGFuZCByZW5kZXJzIGxpa2UgdGhpczogVGVzdCA8LS0-IEQuIiwibWVybWFpZCI6IntcbiAgXCJ0aGVtZVwiOiBcImRlZmF1bHRcIlxufSIsInVwZGF0ZUVkaXRvciI6ZmFsc2UsImF1dG9TeW5jIjp0cnVlLCJ1cGRhdGVEaWFncmFtIjpmYWxzZX0)

**Comment by dwhelan** [comment]
It works with `graph` now in `develop` branch:

<img width="187" alt="image" src="https://user-images.githubusercontent.com/17627/193484239-b1076395-c211-4242-88d9-e8287b1060f7.png">

I believe this could be closed on the next release after `9.1.7`.  Below is how it is rendered `9.1.7` or earlier:
```mermaid
graph LR
    A>Test];
    A <--> B;

    C[Test];
    C <--> D;
```
